### ISSUE TITLE
Jaws/Edge doesnot read the invisible text with Aria-label

### ISSUE THREAD

**Issue by divjose** [FreedomScientific/VFO-standards-support#126]
### Summary

>  aria-label is not read in Edge/Jaws Combo. 
> 
> Example:
> Description:
With the aria-label attribute, authors can pass a text value to the attribute and read it with an associated visible non-clickable element. This gives the screen reader users more contextual information. For example in the Mastery by Standard page we want the score to be read as, 'Class Score 100%, Mastered' where the text displayed is '100%'.

Problem Statement:
Edge/Jaws does not support aria-label on non-clickable elements like span. 

### Expected result
> <span aria-label="Class Score 100%, Mastered">100%</span>
Jaws should read the aria-label tag ignoring the displayed text.  

### Actual result
> Only 100% is read

### Example

>We want the score to be read as, 'Class Score 100%, Mastered' whereas the text displayed is '100%'. This is so to provide contextual information to the non-sighted users. This tabular data is rendered using divs and span. 

The a11y best practice is to use aria-label on the span that needs to convey contextual information. 
<span aria-label="Class Score 100%, Mastered">100%</span>

**Comment by stevefaulkner** [comment]
Given that Edge in its current state [will not continue to be developed](https://twitter.com/stevefaulkner/status/1070761434450141185) closing this as won't fix
